src/native-common/AccessibilityUtil.ts

```diff
@@ -9,8 +9,15 @@
 */
 
 import _ = require('./lodashMini');
+import assert = require('assert');
+import React = require('react');
+import RN = require('react-native');
 
 import { AccessibilityUtil as CommonAccessibilityUtil } from '../common/AccessibilityUtil';
+import AndroidAccessibilityUtil from '../android/AccessibilityUtil';
```

Comment:
Why are you importing platform-specific files within a common file? This is an anti-pattern and will create dependency cycles. Platform-specific files should import native-common, not the other way around.